### ISSUE TITLE
Add slight outline to panel items

### ISSUE THREAD

**Issue by arvigeus** [pop-os/shell#472]
When I set the panel to transparent, items get lost in lighter wallpapers. Example: https://imgur.com/a/kQsWXwP (the right items are hovered by mouse). I've seen other themes having it, it makes for much better experience.

**Comment by jmmaranan** [comment]
I do not think this is a pop-shell issue. Can you describe how you are setting the panel as transparent?

**Comment by arvigeus** [comment]
I am using [Transparent Top Panel](https://extensions.gnome.org/extension/2878/transparent-top-panel/). It works with Nordic theme
